cve_bin_tool/cvedb.py

```diff
@@ -79,11 +79,11 @@ def __init__(
         self.dbpath = self.cachedir / DBNAME
         self.connection: sqlite3.Connection | None = None
 
-        self.data = []
+        self.data: list[Any] = []
```

Comment:
```suggestion
        self.data: List[Any] = []
```
You might also need to change this one (and the one below) to be `List` (imported from typing if needed) instead of lower-case `list`.  that's just a python < 3.9 support thing.

Reply:
Yeah, I tried to change it to List from list. But as soon as I run pre-commit. It just automatically change to list and starts giving me an error in this line "from typing import List" and it says, "List is imported but not used". I tried it many times, but pre-commit is not letting me do it.

Reply:
Hm, very odd.  I haven't seen black or flake8 make that kind of code change, but maybe it's pyupgrade?  Since I just had it pointed out to me that we could use `list` in conjunction with an `import __future__` line instead, we might switch to that *anyhow*.  Given that it's the last day of October and I'm guessing you'd like hacktoberfest credit, I'll go ahead and make that tweak and see if we can get this merged today.